spec/__snapshots__/view.test.js.snap

```diff
@@ -0,0 +1,12 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Please read the following post, and all of the comments on each answer. Then, let's discuss what we think our team should do moving forward:

https://stackoverflow.com/questions/53230363/how-to-gitignore-snapshot-folders

Reply:
I think the answer is on Jest's documentation: https://jestjs.io/docs/en/snapshot-testing.html#should-snapshot-files-be-committed - lemme know what you think

Reply:
TLDR: We should be committing snapshots